File: archive/maximum-number-of-words-you-can-type.js
const canBeTypedWords = function(text, brokenLetters) {
  brokenLetters = brokenLetters.split('')
  text = text.split(' ')

  let ans = 0
  text.forEach(word => {
    let ok = true
    word.split('').forEach(l => {
      if (brokenLetters.includes(l)) {ok = false}
    })

    if (ok) {ans++}
  })

  return ans
};

console.log(canBeTypedWords("hello world", "ad"))

File: maximum-number-of-words-you-can-type.go
func canBeTypedWords(text string, brokenLetters string) int {
    broken := map[byte]bool{}

    for i, _ := range brokenLetters {
        broken[brokenLetters[i]] = true
    }

    splitted := strings.Split(text, " "); ans:=0
    for _, word := range splitted {
        isOk:=true
        for i:=0; i < len(word); i++ {
            if _, isHas := broken[word[i]]; isHas {
                isOk = false
            }
        }
        if isOk {ans++}
    }

    return ans
}